src/main/fc/settings.yaml

```diff
@@ -90,6 +90,9 @@ tables:
   - name: smartport_fuel_unit
     values: ["PERCENT", "MAH", "MWH"]
     enum: smartportFuelUnit_e
+  - name: platform_type
+    values: ["MULTIROTOR", "AIRPLANE", "HELICOPTER", "TRICOPTER", "ROVER", "BOAT"]
```

Comment:
Yay! I like that! Maybe VTOL's should have separate platform types?

Reply:
That would make sense, right?